syntax = "proto3";
import "common.proto";
import "google/protobuf/empty.proto";
package ms2gs;
option cc_generic_services = true;

message EnterGameRequest
{
  uint64 player_id = 1;
  SceneInfo scenes_info = 2;
  uint64 conn_id = 3;
}

message EnterGameRespone
{
}

service Ms2gService {
  rpc EnterGame (EnterGameRequest) returns (EnterGameRespone);
}